portfolio/src/main/webapp/comments.html

```diff
@@ -50,6 +50,10 @@ <h1 class="header">Comments</h1>
         <input type="submit">
       </form>
 
+      <p>
+        Comments shown below with sentiment analysis between -1 and 1. Closer to
+        1 is a more positive comment, while closer to -1 is a more negative one.
+      </p>
```

Comment:
maybe convert to an integer between 1 and 5? Then you can say this is a 3/5 comment?

Reply:
Oh I like that, I've changed the scale to 0 to 10 (the math is a little easier to work with by adding 1 and multiplying by 5).